qcelemental/testing.py

```diff
@@ -102,10 +102,15 @@ def compare_values(expected,
         if expected is None and computed is None:
             return return_handler(True, label, pass_message, return_message, quiet)
 
+    if expected.dtype == np.complex:
```

Comment:
We need `np.iscomplexobj(expected)` here. Otherwise `complex64` etc is incorrectly typed.

Reply:
Good to know there's such a function in NumPy! Let's see what Travis has to say now.